pkg/override/override.go

```diff
@@ -0,0 +1,139 @@
+package override
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/josephburnett/jd/v2"
+	"github.com/samber/oops"
+	"go.yaml.in/yaml/v4"
+)
+
+// Config represents the override configuration file
+type Config struct {
+	Patches []PatchEntry `yaml:"patches"`
+}
+
+// PatchEntry represents a single patch entry
+type PatchEntry struct {
+	Target string `yaml:"target"` // Path suffix to match (e.g., "ghsa/2025/11/GHSA-xxxx.json")
+	Diff   string `yaml:"diff"`   // Path to jd diff file (relative to overrides dir)
+}
+
+// Patches holds loaded override configuration
+type Patches struct {
+	entries      []PatchEntry
+	overridesDir string
+}
+
+// Patch represents a matched patch that can be applied to content
+type Patch struct {
+	diff jd.Diff
+}
+
+// Load reads config.yaml from the given directory
+func Load(overridesDir string) (*Patches, error) {
+	eb := oops.With("overrides_dir", overridesDir)
+
+	configPath := filepath.Join(overridesDir, "config.yaml")
+	f, err := os.Open(configPath)
+	if err != nil {
+		return nil, eb.Wrapf(err, "failed to open config file")
+	}
+	defer f.Close()
+
+	var cfg Config
+	if err := yaml.NewDecoder(f).Decode(&cfg); err != nil {
+		return nil, eb.Wrapf(err, "failed to parse config.yaml")
+	}
+
+	patches := &Patches{
+		entries:      make([]PatchEntry, 0, len(cfg.Patches)),
+		overridesDir: overridesDir,
+	}
+
+	for _, p := range cfg.Patches {
+		eb := eb.With("target", p.Target, "diff", p.Diff)
```

Comment:
nit:
```suggestion
		eb := eb.With("target", p.Target).With("diff", p.Diff)
```

Reply:
As [documented](https://pkg.go.dev/github.com/samber/oops#With), `With` can take a list of KVs. Do you prefer repeating `With` for readability?
https://go.dev/play/p/30kFZS8V6o1

Reply:
> Do you prefer repeating With for readability?

Yes, I think it’s easier to read the code when pairs are separated.

Reply:
Fixed in b47ea17.